### DISCUSSION TITLE
Custom integration

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [spatie/ray#322] [Q&A] [answered]
I would like to integrate Ray into October CMS. I've created my own little package with currently only one function. 

https://github.com/codecyclernl/oc-ray-plugin/tree/master

Will it be ever possible to use the same function `ray()` in custom integrations? I see the default Ray package checks if one of the integrations is installed.

**Comment by freekmurze** [comment, marked as answer]
Feel free to send a PR that would enable the ray() function to work for your package. Also, feel free to PR the docs of your package to our docs!

**Reply by SebastiaanKloos** [reply]
Thank you! Will definitely do 👍
